test/common/ws/server_process.js

```diff
@@ -40,9 +40,9 @@ function WSServer(headers) {
 		secretKey: 'liskSecretKey',
 		headers,
 		perMessageDeflate: false,
-		pingInterval: 5000,
-		pingTimeout: 60000,
-		processTermTimeout: 10000,
+		pingInterval: 500,
+		pingTimeout: 600,
```

Comment:
I don't think we want the socket to ever timeout during the tests so we should make the `pingTimeout` higher. I think `1100` would be a better value here; that means that it won't timeout/disconnect the socket unless it misses two full pings.

Reply:
Actually maybe a `10000` is better for `pingTimeout`; if the testsuite is running on a weak machine, it could lag so it's good to give extra margin of safety.

Reply:
I've reverted the three parameters as they were before.